package.json

```diff
@@ -34,6 +34,7 @@
   "devDependencies": {
     "babel-cli": "^6.10.1",
     "babel-eslint": "^6.0.5",
+    "babel-plugin-transform-regenerator": "^6.21.0",
```

Comment:
Since we've removed the code that used the generator and we're not using any at this time, can we remove this for now? We could always bring it in should we want to implement a generator at some point.